wsmaster/che-core-api-workspace/src/test/resources/devfile/schema_test/devfile/devfile_v2-1-0-alpha_with_invalid_plugin_definition.yaml

```
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2012-2018 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+schemaVersion: 2.1.0-alpha
+metadata:
+  name: test-project
+  attributes:
+    type: workspace
+projects:
+  - name: test-project
+    git:
+      remotes:
+        origin: https://github.com/Test/Project
+components:
+  - plugin:
+      name: java language server
+      id: redhat/java11/0.57.0
```

end file empty lines are missing here an above
```suggestion
      id: redhat/java11/0.57.0

```